pkg/storage/trb_advice_letter_recommendation.go

```diff
@@ -212,7 +215,8 @@ func (s *Store) UpdateTRBAdviceLetterRecommendation(ctx context.Context, recomme
 // DeleteTRBAdviceLetterRecommendation deletes an existing TRB advice letter recommendation record in the database
 func (s *Store) DeleteTRBAdviceLetterRecommendation(ctx context.Context, id uuid.UUID) (*models.TRBAdviceLetterRecommendation, error) {
 	stmt, err := s.db.PrepareNamed(`
-		DELETE FROM trb_advice_letter_recommendations
+		UPDATE trb_advice_letter_recommendations
+		SET deleted_at = NOW(), position_in_letter = NULL
```

Comment:
Rather than `NOW()`, we should probably use `CURRENT_TIMESTAMP` to be consistent with other code; from [the Postgres docs](https://www.postgresql.org/docs/14/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT), it looks like they're functionally identical.

Reply:
37a62e01a